LLMonFHIR/FHIR Display/ResourceChatView.swift

```diff
@@ -0,0 +1,71 @@
+//
+// This source file is part of the Stanford LLM on FHIR project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziOpenAI
+import SwiftUI
+
+
+struct ResourceChatView: View {
+    @Environment(\.dismiss) var dismiss
+    @EnvironmentObject var openAPIComponent: OpenAIComponent<FHIR>
+    @EnvironmentObject var fhirResourceSummary: FHIRResourceSummary<FHIR>
+    @EnvironmentObject var fhirStandard: FHIR
+
+    @State var chat: [Chat]
+    @State var gettingAnswer = false
+    
+    let title: String
+
+    var body: some View {
+        NavigationStack {
+            ChatView($chat, disableInput: $gettingAnswer)
+                .navigationTitle(title)
+                .toolbar {
+                    ToolbarItem(placement: .cancellationAction) {
+                        Button("FHIR_RESOURCES_CHAT_CANCEL") {
+                            dismiss()
+                        }
+                    }
+                }
+                .onChange(of: chat) { _ in
+                    if !gettingAnswer {
+                        getAnswer()
+                    }
+                }
+        }
+    }
+    
```

Comment:
Very minor change: We normally add two empty lines between different types of elements in a type, e.g. two empty lines between stored properties and computed properties in that case.
```suggestion


```